data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py

```diff
@@ -0,0 +1,693 @@
+
+# + [markdown] papermill={"duration": 0.02327, "end_time": "2021-02-02T22:30:32.708257", "exception": false, "start_time": "2021-02-02T22:30:32.684987", "status": "completed"} tags=[]
+# # QA query 1
+
+# + papermill={"duration": 4.105203, "end_time": "2021-02-02T22:30:36.813460", "exception": false, "start_time": "2021-02-02T22:30:32.708257", "status": "completed"} tags=[]
+#newcdr='2020q4r1_combined_release'
+query = f'''
+
+SELECT
+
+d.observation_id AS observation_id,
+
+i.observation_id AS observation_deid,
+
+m.research_id AS person_id,
+
+m.person_id AS research_id,
+
+d.observation_date AS date_d,
+
+i.observation_date AS date_i,
+
+m.shift
+
+FROM
+
+`{project_id}.pipeline_tables.pid_rid_mapping` m
+
+JOIN
+
+`{project_id}.{com_cdr}.observation` i
+
+ON
+
+m.person_id = i.person_id
+
+JOIN
+
+`{project_id}.{new_cdr}.observation` d
+
+ON
+
+d.observation_id = i.observation_id
+
+
+limit 5
+
+
+'''
+pd.read_gbq(query, dialect='standard')
+
+# + [markdown] papermill={"duration": 0.023633, "end_time": "2021-02-02T22:30:36.860798", "exception": false, "start_time": "2021-02-02T22:30:36.837165", "status": "completed"} tags=[]
+# ##  QA  query 2
+
+# + papermill={"duration": 2.136748, "end_time": "2021-02-02T22:30:39.044867", "exception": false, "start_time": "2021-02-02T22:30:36.908119", "status": "completed"} tags=[]
+query = f'''
+
+
+
+
+SELECT
+
+d.observation_id AS observation_id,
+
+d.observation_source_value AS observation_source_value,
+
+i.observation_id AS observation_deid,
+
+m.research_id AS person_id,
+
+m.person_id AS research_id,
+
+d.observation_date AS date_d,
+
+i.observation_date AS date_i,
+
+m.shift
+
+FROM
+
+`{project_id}.pipeline_tables.pid_rid_mapping` m
+
+JOIN
+
+`{project_id}.{com_cdr}.observation` i
+
+ON
+
+m.person_id = i.person_id
+
+JOIN
+
+`{project_id}.{new_cdr}.observation` d
+
+ON
+
+d.observation_id = i.observation_id
+
+Where d.observation_source_value LIKE 'PIIBirthInformation_BirthDate'
+
+limit 5
+
+'''
+pd.read_gbq(query, dialect='standard')
+# -
+
+# # query 3
+
+# + papermill={"duration": 2.136748, "end_time": "2021-02-02T22:30:39.044867", "exception": false, "start_time": "2021-02-02T22:30:36.908119", "status": "completed"} tags=[]
+query = f'''
+
+ SELECT
+
+d.observation_period_id AS observation_period_id,
+
+m.research_id AS person_id,
+
+m.person_id AS research_id,
+
+d.observation_period_start_date AS date_d,
+
+i.observation_period_start_date  AS date_i,
+
+m.shift,
+
+FROM
+
+`{project_id}.pipeline_tables.pid_rid_mapping` m
+
+JOIN
+
+`{project_id}.{com_cdr}.observation_period` i
+
+ON
+
+m.person_id = i.person_id
+
+JOIN
+
+`{project_id}.{new_cdr}.observation_period` d
+
+ON
+
+d.observation_period_id = i.observation_period_id
+
+limit 5
+
+'''
+pd.read_gbq(query, dialect='standard')
+
+# + [markdown] papermill={"duration": 0.023649, "end_time": "2021-02-02T22:30:39.115495", "exception": false, "start_time": "2021-02-02T22:30:39.091846", "status": "completed"} tags=[]
```

Comment:
These papermill execution logs should also not be committed to the repo. I think they should be able to be cleared, but let me know if you can't.

Reply:
i am not sure about this part. basically, i use Jupiter notebook to edit and py file is saved in the fly. i could not see these execution logs in the notebook but do see this in the py file. not really sure what I can do.

Reply:
Hmm, I think I see what's happening. The notebook runner updates the input notebook if no `--output_path` is passed. To avoid this , you could pass a `--output_path` with a different name than your input notebook.

Reply:
Can you pull your clean, un-run version of your jupytext notebook from anywhere?

Reply:
yes, done. now i understood what you were saying. thanks.